bundle/Core/Exception/InvalidStateException.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\IbexaScheduledVisibilityBundle\Core\Exception;
+
+use Exception;
+use Ibexa\Contracts\Core\Repository\Values\Content\Content;
+
+use function sprintf;
+
+final class InvalidStateException extends Exception
+{
+    public function __construct(Content $content, ?Exception $previous = null)
+    {
+        $message = sprintf("Content '%s' with id #%d is not valid for scheduled visibility update.", $content->getName(), $content->getId());
```

Comment:
We'll still need to change this to `$content->id` to support Ibexa 4.5.

Reply:
fixed in 823dd0daa3d1ddef28adcd4701fd68507c0c02a6